### PR TITLE
Create ui-c8y-1021-4-3-only-error-from-auth-external-server-are-shown-in-additional-modal.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-60547](https://cumulocity.atlassian.net/browse/MTM-60547)
Original PR: [7594](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7594)

[MTM-60547]: https://cumulocity.atlassian.net/browse/MTM-60547?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1021-4-3-only-error-from-auth-external-server-are-shown-in-additional-modal.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: only error from auth external server are shown in additional modal (#6937) [GRAFT][release/cd] (#7594)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1021-4-3-only-error-from-auth-external-server-are-shown-in-additional-modal.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Improved error handling for external authentication servers
```

File: content/change-logs/platform-services/ui-c8y-1021-4-3-only-error-from-auth-external-server-are-shown-in-additional-modal.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60547
+version: 1021.4.3
+---
+only error from auth external server are shown in additional modal (#6937) [GRAFT][release/cd] (#7594)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1021-4-3-only-error-from-auth-external-server-are-shown-in-additional-modal.md:17]: The Claude AI suggested to use the following body:
```suggestion
Previously, errors from external authentication servers were not always displayed to the user, leading to confusion when authentication failed without any visible indication. With this change, all errors originating from external authentication servers are now consistently shown in a dedicated error modal dialog. This improves the user experience by providing clear feedback on authentication failures, making it easier for users to identify and potentially resolve any issues with their external authentication setup.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@mmed-c8y @kowa-c8y Please check this PR

File: content/change-logs/platform-services/ui-c8y-1021-4-3-only-error-from-auth-external-server-are-shown-in-additional-modal.md (hunk 2 above)
Comment (kowa-c8y) [inline content/change-logs/platform-services/ui-c8y-1021-4-3-only-error-from-auth-external-server-are-shown-in-additional-modal.md]:
```suggestion
Previously, errors from external authentication servers were not always displayed to the user, causing confusion when authentication failed without any visible indication. With this update, all errors originating from external authentication servers are now consistently shown in a dedicated error modal dialog. Additionally, errors resulting from insufficient permissions within the platform are now presented clearly and in a user-friendly manner on the login page.
```

Comment (kowa-c8y) [inline content/change-logs/platform-services/ui-c8y-1021-4-3-only-error-from-auth-external-server-are-shown-in-additional-modal.md]:
```suggestion
Enhanced display of error messages for external authentication and platform permissions
```

**Comment by kowa-c8y** [comment]
I added my suggested changes @mmed-c8y @BeateRixen  please verify it

File: content/change-logs/platform-services/ui-c8y-1021-4-3-only-error-from-auth-external-server-are-shown-in-additional-modal.md (hunk 1 above)
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1021-4-3-only-error-from-auth-external-server-are-shown-in-additional-modal.md]:
```suggestion
title: Enhanced display of error messages for external authentication and platform permissions
```

File: content/change-logs/platform-services/ui-c8y-1021-4-3-only-error-from-auth-external-server-are-shown-in-additional-modal.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60547
+version: 1021.4.3
+---
+Enhanced display of error messages for external authentication and platform permissions
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1021-4-3-only-error-from-auth-external-server-are-shown-in-additional-modal.md]:
```suggestion
Previously, errors from external authentication servers were not always displayed to the user, causing confusion when authentication failed without any visible indication. With this update, all errors originating from external authentication servers are now consistently shown in a dedicated error modal dialog. Additionally, errors resulting from insufficient permissions within the platform are now presented clearly and in a user-friendly manner on the login page.
```

File: content/change-logs/platform-services/ui-c8y-1021-4-3-only-error-from-auth-external-server-are-shown-in-additional-modal.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60547
+version: 1021.4.3
+---
+Previously, errors from external authentication servers were not always displayed to the user, causing confusion when authentication failed without any visible indication. With this update, all errors originating from external authentication servers are now consistently shown in a dedicated error modal dialog. Additionally, errors resulting from insufficient permissions within the platform are now presented clearly and in a user-friendly manner on the login page.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1021-4-3-only-error-from-auth-external-server-are-shown-in-additional-modal.md]:
```suggestion
Previously, errors from external authentication servers were not always displayed to the user, causing confusion when authentication failed without any visible indication. With this update, all errors originating from external authentication servers are now consistently shown in a dedicated error dialog. Additionally, errors resulting from insufficient permissions within the platform are now presented clearly and in a user-friendly manner on the login page.
```

**Review by BeateRixen — APPROVED** [review]

File: content/change-logs/platform-services/ui-c8y-1021-4-3-only-error-from-auth-external-server-are-shown-in-additional-modal.md (hunk 2 above)
Comment (JanePorter) [reply content/change-logs/platform-services/ui-c8y-1021-4-3-only-error-from-auth-external-server-are-shown-in-additional-modal.md]: @kowa-c8y I like the proposed change

Comment (BeateRixen) [reply content/change-logs/platform-services/ui-c8y-1021-4-3-only-error-from-auth-external-server-are-shown-in-additional-modal.md]: @kowa-c8y Please approve, then I can merge

**Review by kowa-c8y — APPROVED** [review]

**Comment by kowa-c8y** [comment]
@BeateRixen  approved :+1:

**Review by saran-govindarajan — APPROVED** [review]

**Comment by kowa-c8y** [comment]
@BeateRixen  Can we merge it?

**Pull request merged**